### PR TITLE
add title and decoration for bot dream response

### PR DESCRIPTION
**Pull request by MahmoudHafezDarwish**

### CODE REVIEW

File: lib/src/chat_bot/ui/chat_bubble.dart [hunk 1]
```diff
@@ -31,23 +31,39 @@ class ChatBubble extends StatelessWidget {
...
+              borderRadius: bubbleDecoration.borderRadius,
+            ),
+            child: chatMessageBuilder.build(message) ??
+                Column(
+                  mainAxisSize: MainAxisSize.min,
+                  children: [
+                    Visibility(
+                      visible: message.title != null,
+                      child: Text(
+                        message.title ?? '',
```
Comment (mohamedalwhaidi) [inline lib/src/chat_bot/ui/chat_bubble.dart:50]: remove nullable

File: lib/src/chat_bot/ui/chat_bubble.dart [hunk 2]
```diff
@@ -31,23 +31,39 @@ class ChatBubble extends StatelessWidget {
...
+              minWidth: decoration.minWidth,
+            ),
+            padding: decoration.padding,
+            decoration: BoxDecoration(
+              color: bubbleDecoration.color,
+              borderRadius: bubbleDecoration.borderRadius,
+            ),
+            child: chatMessageBuilder.build(message) ??
+                Column(
+                  mainAxisSize: MainAxisSize.min,
```
Comment (mohamedalwhaidi) [inline lib/src/chat_bot/ui/chat_bubble.dart:45]: cross default: center?

**Review by mohamedalwhaidi — CHANGES_REQUESTED** [review]

File: lib/src/chat_bot/ui/chat_bubble.dart (hunk 1 above)
Comment (MahmoudHafezDarwish) [reply lib/src/chat_bot/ui/chat_bubble.dart:50]: done

File: lib/src/chat_bot/ui/chat_bubble.dart (hunk 2 above)
Comment (MahmoudHafezDarwish) [reply lib/src/chat_bot/ui/chat_bubble.dart:45]: I put it Cross.center

**Review by mohamedalwhaidi — APPROVED** [review]

**Pull request merged**